lightwood/api/predictor.py

```diff
@@ -177,15 +173,18 @@ def type_map(col_name):
         mixer_params = {}
 
         if 'mixer' in self.config:
-            mixer_class = self.config['mixer']['class']
+            if 'class' in  self.config['mixer']:
+                mixer_class = self.config['mixer']['class']
             if 'attrs' in  self.config['mixer']:
                 mixer_params = self.config['mixer']['attrs']
         else:
             mixer_class = NnMixer
```

Comment:
mixer_class = NnMixer
mixer_params = {}

if 'mixer' in self.config: ...

Reply:
Sure